### ISSUE TITLE
save_resume doesn't save all parameters

### ISSUE THREAD

**Issue by MochizukiShinichi** [VowpalWabbit/vowpal_wabbit#3790]
#### Describe the bug
save_resume doesn't save learning_rate , l1, l2 and some parameters.

#### To Reproduce
In python (same behavior in command line)
```
from vowpalwabbit import pyvw
model = pyvw.vw(learning_rate=0.01, boosting=100, final_regressor="test_model2.vw", save_resume=True)
model.finish()
pyvw.vw(initial_regressor="test_model2.vw", save_resume=True).get_arguments()
```

#### Expected behavior
outputs should contain learning_rate=0.01 

#### Observed Behavior
output from above command
`'--bit_precision 18 --boosting 100 --initial_regressor test_model2.vw --save_resume
`
#### Environment
What version of VW did you use? 8.11.0 (git commit: 6b45001e8) with python 
What OS or language did you use? debian10

**Comment by jackgerrits** [comment]
Learning rate and power_t are stored in the model as of version 9.0.0. Fixed in this PR: #3658
